webpack.config.js

```diff
@@ -26,6 +26,30 @@ module.exports = async (env, options) => {
       taskpane: ['react-hot-loader/patch', './src/taskpane/index.tsx'],
       commands: './src/commands/commands.ts',
     },
+    optimization: {
+      splitChunks: {
+        chunks: 'all',
+        minSize: 20000,
+        maxSize: 50000,
```

Comment:
I'm a bit unsure about the maxSize. The taskpane is now split in over 100 chunks. While this is more performant during local development, I suspect this actually performs worse if you add real latency when deployed to a server. However, I'm not sure how to test it at the moment. Should we wait until I've fixed the deployment to azure (hopefully in the next few days)?